lighthouse-core/config/default.js

```diff
@@ -615,6 +620,7 @@ module.exports = {
         {"id": "first-meaningful-paint", "weight": 5},
         {"id": "speed-index-metric", "weight": 1},
         {"id": "estimated-input-latency", "weight": 1},
+        {"id": "time-to-firstbyte", "weight": 1},
```

Comment:
going forward we're only going to be scoring the actual metrics in performance, so we can set this weight to 0

nit: could you move this down below with the other 'informative' audits so we don't split the metrics